synapse/tests/test_model_inet.py

```diff
@@ -2754,3 +2755,13 @@ async def test_model_inet_egress(self):
             self.eq(nodes[0].get('client'), 'tcp://1.2.3.4')
             self.eq(nodes[0].get('client:ipv4'), 0x01020304)
             self.eq(nodes[0].get('client:ipv6'), '::1')
+
+    async def test_cpe_scrape_one_to_one(self):
```

Comment:
I put this test in the wrong file :laughing:  lets move it to test_model_infotech